commit a3b1d2c9d8249d839942fd6a44df25c0742bbfd6 (refs/changes/74/5174/3)
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2018-12-14T21:26:59+00:00 (10 months ago)
    
    TOOLS-2117 sdcnode updates for November 2018 node.js security releases
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index 305da56..9b821a9 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright 2016, Joyent, Inc.
+# Copyright 2018, Joyent, Inc.
 #
 
 #
@@ -32,7 +32,7 @@ JSSTYLE_FLAGS	= -f tools/jsstyle.conf
 ESLINT_FILES	= $(JS_FILES)
 
 ifeq ($(shell uname -s),SunOS)
-	NODE_PREBUILT_VERSION =	v6.14.3
+	NODE_PREBUILT_VERSION =	v6.15.1
 	NODE_PREBUILT_TAG =	zone
 	NODE_PREBUILT_IMAGE =	18b094b0-eb01-11e5-80c1-175dac7ddf02
 endif
